github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -140,6 +140,39 @@ periodics:
     - name: gcs
       secret:
         secretName: gcs
+- name: periodic-kubevirt-e2e-k8s-1.19-sig-network
+  annotations:
+    testgrid-create-test-group: "false"
+  interval: 8h
+  decorate: true
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+  extra_refs:
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: master
+    work_dir: true
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+      - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+        command: [ "/usr/local/bin/runner.sh", "/bin/sh", "-c" ]
+        args:
+          - |
+            export TARGET=k8s-1.19-sig-network
```

Comment:
Can you pls move the env var definition to a `env` entry? Like here https://github.com/kubevirt/project-infra/blob/master/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml#L332-L334 Once this is done it would be better to move the `automation/test.sh` call to the `command` array, so that it looks like:
```
command:
  - "/usr/local/bin/runner.sh"
  - "/bin/sh"
  - "-c"
  - "automation/test.sh"
```
and remove the `args` entry